src/modules/LanItems.rb

```diff
@@ -737,8 +737,9 @@ def write
 
       LanItems.WriteUdevRules if !Stage.cont && InstallInfConvertor.instance.AllowUdevModify
 
+      # FIXME: remove this code
       # FIXME: hack: no "netcard" filter as biosdevname names it diferently (bnc#712232)
-      NetworkInterfaces.Write("")
+      # NetworkInterfaces.Write("")
```

Comment:
ok we usually don't keep old code as comments, so if there is a reason for it it would deserve a comment.